games-arcade/wop/wop-0.4.3-r2.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+EAPI=6
+inherit eutils toolchain-funcs
+
+MY_DATA_V="2005-12-21"
+MY_DATA_P="${PN}data-${MY_DATA_V}"
+DESCRIPTION="Worms of Prey - A multi-player, real-time clone of Worms"
+HOMEPAGE="http://wormsofprey.org/"
+SRC_URI="http://wormsofprey.org/download/${P}-src.tar.bz2
+	http://wormsofprey.org/download/${MY_DATA_P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~x86-fbsd"
+IUSE=""
+
+RDEPEND="media-libs/libsdl
+	media-libs/sdl-net
+	media-libs/sdl-image
+	media-libs/sdl-mixer
+	media-libs/sdl-ttf"
+DEPEND="${RDEPEND}
+	x11-misc/makedepend"
+
+MY_DATA_S=${WORKDIR}/${MY_DATA_P}
+
+src_prepare() {
+	default
+
+	# patch global woprc with the correct data files location and install it
+	sed -i \
+		-e "s:^data =.*$:data = /usr/share/${PN}:" \
+		woprc \
+		|| die "sed failed"
+
+	eapply -p0 "${FILESDIR}"/${P}-Makefile.patch
+	eapply -p0 "${FILESDIR}"/${P}-gcc43.patch
+	eapply -p1 "${FILESDIR}"/${P}-gcc6.patch
```

Comment:
please put these in PATCHES and fix them, i.e., add pseudo git `a/` `b/` to paths to make them -p1 patches